src/components/FileTreeNode/index.spec.tsx

```diff
@@ -129,7 +131,7 @@ describe(__filename, () => {
     treefoldRenderProps = {},
   }) => {
     const externalVersion = fakeVersion;
-    const store = createStoreWithVersion(externalVersion);
+    const store = createStoreWithVersion({ makeCurrent: true });
```

Comment:
This needs to still pass `version: externalVersion` since it has an explicit dependency on `externalVersion`. Otherwise, the test relies on the default.

Reply:
I thought since `externalVersion = fakeVersion` , So it would be fine to use `makeCurrent : true` here. Anyways Thanks for explaining, will update the PR.